src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,63 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    
+    <div *ngIf="childrendataSource.data.length > 0 && !isLoading" class="table-wrapper">
+      <table mat-table [dataSource]="childrendataSource"  >
+        <tr hidden="true">
+          <th scope="col" >Child Name </th>
+          <th scope="col"  >DateOfBirth</th>
+          <th scope="col" >AfterBirthdayAge</th>
+          
+        </tr>
+        <ng-container matColumnDef="name">
+          <td mat-cell *matCellDef="let child">{{ child.name}} 
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="dateOfBirth">
+          <td mat-cell *matCellDef="let child">{{child.dateOfBirth | date:'dd-MM-YYYY' }} </td>
+        </ng-container>
+
+        <ng-container matColumnDef="age">
+          <td mat-cell *matCellDef="let child">{{child.age +1}}</td>
+        </ng-container>
+
+    
+<!--
+  TODO
+  - replace this by a table similar to `AttendanceWeekDashboard` (for more info see https://v11.material.angular.io/components/table/overview)
+  - display the child using the `ChildBlockComponent`
+  - display the birthday
+  - display the age the child will have after its birthday
+   
+-->
+           <app-child-block
```

Comment:
This is an example how you can use the `<app-child-block>` component. Please use this in the table and then remove it from here.